pkg/rtc/participant.go

```diff
@@ -1708,6 +1704,14 @@ func (p *ParticipantImpl) GetPendingTrack(trackID livekit.TrackID) *livekit.Trac
 	return nil
 }
 
+func (p *ParticipantImpl) GetICEConnectionDetails() []*types.ICEConnectionDetails {
+	return p.TransportManager.GetICEConnectionDetails()
+}
+
+func (p *ParticipantImpl) HasConnected() bool {
```

Comment:
adding a new interface method. useful to know if Participant was ever successful